source/_integrations/zwave_js.markdown

```diff
@@ -549,81 +523,98 @@ You can also visit the `#zwave` channel on [our discord](/join-chat/).
+If you are using Home Assistant Container, Home Assistant Core, or you don't want to use the built-in Z-Wave JS Server add-on, you will need to run the Z-Wave JS server yourself, to which the Z-Wave JS integration will connect.
+
+### Running [Z-Wave JS Server](https://github.com/zwave-js/zwave-js-server).
 
-### Requirements
+This application provides the connection between your Z-Wave USB stick and Home Assistant. The Home Assistant Z-Wave JS integration connects to this server via a websocket connection. You need to run this Z-Wave server before you can use the integration.
 
-Controlling your Z-Wave network using the Z-Wave JS integration has the following requirements:
+There are multiple ways to run this server:
 
-1. Run [Z-Wave JS Server](https://github.com/zwave-js/zwave-js-server). This application provides the connection between your Z-Wave USB stick and Home Assistant. This server runs separately from Home Assistant so your Z-Wave mesh will keep running if you restart or stop Home Assistant. The Home Assistant Z-Wave JS integration connects to this server via a websocket connection. You need to run this Z-Wave server before you can use the integration.
+**Option 1: The official Z-Wave JS add-on, as described above**
 
-2. [Supported Z-Wave dongle](/docs/z-wave/controllers/#supported-z-wave-usb-sticks--hardware-modules). The Z-Wave controller dongle should be connected to the same host as where the Z-Wave JS server is running. In the configuration for the Z-Wave server you need to provide the path to this stick. It's recommended to use the `/dev/serial-by-id/yourdevice` version of the path to your stick, to make sure the path doesn't change over reboots. The most common known path is `/dev/serial/by-id/usb-0658_0200-if00`.
+_This option is only available for Home Assistant OS (the recommended installation type) and Home Assistant Supervised installations._
 
-3. A **network key**. This key is used in order to connect securely to compatible devices. The network key consists of 32 hexadecimal characters, for example `2232666D100F795E5BB17F0A1BB7A146` (do not use this one, pick a random one). Without a network key security enabled devices cannot be added securely and will not function correctly. You must provide this network key in the configuration part of the Z-Wave JS Server. For new installations, a unique default key will be auto generated for you. TIP: You could use a site like random.org to create your own random network key. Make sure that you keep a backup of this key in a safe place. You will need to enter the same key to be able to access the securely paired devices.
+This add-on can only be configured via the built-in Z-Wave JS control panel in Home Assistant. While that control panel gains new features with every release, some advanced configurations still require the Zwavejs2Mqtt UI.
```

Comment:
This is a very common question, though. People ask all the time what the benefit of the new integration is. Or they get upset they have to change because they want to do something advanced and didn't know to investigate if it was supported. It changes but that's why I left it generic. Do you really think we shouldn't acknowledge that at this point there are still some limitations in the built-in UI?

Reply:
How about this, I dropped that line but changed the zjs2mqtt one to read: "The Z-Wave network can be configured via the built-in Z-Wave JS control panel in Home Assistant and alternatively via the Z-Wave JS control panel built into Zwavejs2Mqtt. It provides you with a full-fledged, attractive and feature-complete UI to manage your Z-Wave nodes and settings, which may support more advanced use cases as development continues on the Z-Wave JS control panel."